pkg/utils/k8sutil.go

```diff
@@ -441,6 +443,15 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 		dpm.Spec.Template.Spec.Containers[0].LivenessProbe = livenessProbe
 	}
 
+	if len(funcObj.Spec.Template.Spec.Containers) > 0 {
```

Comment:
I think https://github.com/kubeless/kubeless/pull/217#discussion_r129995972 got lost, and still needs to be addressed here.